src/money.js

```diff
@@ -21,6 +21,10 @@ export default class Money {
     return dollars * 100 + cents;
   }
 
+  static format(amount) {
+    return amount.toFixed(2).replace(/\d(?=(\d{3})+\.)/g, "$&,");
```

Comment:
Any reason not to do this with active support? https://api.rubyonrails.org/classes/ActionView/Helpers/NumberHelper.html#method-i-number_with_delimiter

Reply:
Because this is JavaScript :)